src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1447,9 +1443,6 @@ void goto_instrument_parse_optionst::instrument_goto_program()
   {
     do_indirect_call_and_rtti_removal();
 
-    // recalculate numbers, etc.
-    goto_model.goto_functions.update();
```

Comment:
Is this actually still needed, seeing as this follows from `do_indirect_call_and_rtti_removal`, not `do_return_removal` ?

Reply:
Well, this commit just reverts that earlier change, I don't think it ever was needed.